pages/profile/content/index.js

```diff
@@ -41,14 +41,14 @@ module.exports = {
   },
   errors: {
     firstName: {
-      required: 'First name is required'
+      required: 'You need to enter a first name.'
     },
     lastName: {
-      required: 'Last name is required'
+      required: 'You need to enter a last name.'
     },
     email: {
       required: 'Email address is required',
-      match: 'Invalid email address'
+      match: 'You need to enter an email address for this person.'
```

Comment:
It looks like you've edited the wrong message here. The one that's edited is for when someone has entered a value, but it's not an email address.

The "there is no value in the field" message is the line above.